### PR TITLE
Site Adjustments (And some Fixes) (MASSIVE MAP UPDATE)

### PR DESCRIPTION
**Pull request by ChromeFoxxity**
## About the Pull Request

Several things are changed, I can't list all of them.

## Why It's Good For The Game

Adds different gameplay, some are unarmored, some are slower. Some also then have different jobs.

## Changelog

:cl:
add: Adds headsets to make communcations easier, adds new armor, adds Recontainment unit room, adds Combat Medic medkits, adds Riot Control metal shields. 
fix: Fixed heavy revolver.
/:cl:

image
image
image
image
image

### CODE REVIEW

File: code/game/jobs/job/security.dm [hunk 1]
```diff
@@ -221,8 +221,8 @@
 	//duties = "<big><b>As the Guard you have more access than a Junior Guard, but do not control them. You are to guard tests and SCP's in the zone you spawned in. If in doubt, ask your Zone or Guard Commander. You also have the duty of guarding the D-Class Cell Blocks. You should not leave your zone under usual SoP.</b></big>"
 	supervisors = "the LCZ Zone Commander"
 	economic_power = 4
-	requirements = list(EXP_TYPE_LCZ = 480)
-	alt_titles = list("LCZ Senior Containment Response Agent", "LCZ Containment Response Sergeant", "LCZ Senior Combat Medic" = /decl/hierarchy/outfit/job/security/lcz_medic, "LCZ Riot Control Sergeant" = /decl/hierarchy/outfit/job/security/lcz_riot, "LCZ Senior Riot Control Agent" = /decl/hierarchy/outfit/job/security/lcz_riot, "LCZ Senior Agent")
+	requirements = list("LCZ Guard" = 480)
```
Comment (TheDarkElites) [inline code/game/jobs/job/security.dm]: Revert this change

Likely caused by GitHub fuckery

**Review by TheDarkElites — CHANGES_REQUESTED** [review]

File: code/game/jobs/job/security.dm (hunk 1 above)
Comment (TheDarkElites) [reply code/game/jobs/job/security.dm]: Specifically the req change not alt titles

File: code/modules/clothing/glasses/hud.dm [hunk 2]
```diff
@@ -62,6 +62,23 @@
 	var/global/list/jobs[0]
 	req_access = list(ACCESS_SECURITY_LVL1)
 
+/obj/item/clothing/glasses/hud/night/security
```
Comment (TheDarkElites) [inline code/modules/clothing/glasses/hud.dm:65]: Sec shouldent have nvgs

**Review by TheDarkElites — CHANGES_REQUESTED** [review]

File: code/modules/clothing/glasses/hud.dm (hunk 2 above)
Comment (ChromeFoxxity) [reply code/modules/clothing/glasses/hud.dm:65]: They are sechud nightvision goggles, I was gonna make them for OMEGA-1.

**Comment by ChuChu42** [comment]
lcz staff cannot get into their own division rooms, open their lockers or use their vendors

ex: response agent, cant enter response agent room or use the gear

**Comment by ChromeFoxxity** [comment]
> lcz staff cannot get into their own division rooms, open their lockers or use their vendors
> 
> ex: response agent, cant enter response agent room or use the gear

That was an accident, but it works due to the division being removed due to development staff wishing for such.

**Comment by bobjension** [comment]
scp5
scp4
scp3
scp2
scp1
scp
i don't intend for the coping to start in this comment section but i'll leave this here

**Comment by Ambrose997** [comment]
Not really sure why you blocked off the class D smelter when smelted ores have never been part of the reasons for escapes, at most they make baseball bats and are a nuisance to the guards. 

Also LCZ medical has no chem dispenser lmao. GL to the doctors relying on the shitty supply in the first aid kit storage and having to run to EZ to get important chems rather than being able to make the on-the-fly. I look forward to both the Class D and LCZ Guard mortality rate increasing after this.

There's other issues which actually make escape far easier than it was before, but for shits and giggles i'm gonna refrain from mentioning them and let this be apparent when it eventually happens lol.

**Comment by ChromeFoxxity** [comment]
> Not really sure why you blocked off the class D smelter when smelted ores have never been part of the reasons for escapes, at most they make baseball bats and are a nuisance to the guards.
> 
> Also LCZ medical has no chem dispenser lmao. GL to the doctors relying on the shitty supply in the first aid kit storage and having to run to EZ to get important chems rather than being able to make the on-the-fly. I look forward to both the Class D and LCZ Guard mortality rate increasing after this.
> 
> There's other issues which actually make escape far easier than it was before, but for shits and giggles i'm gonna refrain from mentioning them and let this be apparent when it eventually happens lol.

Good for you, the chemistry lab is only a security precaution, and it isn't fully sealed off, it's still technically able to be broken into.

**Comment by Ambrose997** [comment]
The chemistry lab doesn't exist. I think you mean the smelter. 

And I think you underestimate how powerful spiders are on this code. The one present in the smelting area that is now walled off will fuck up anyone who tries with ease.

**Comment by Ambrose997** [comment]
Secondly, i'm unsure of how you expect it to be broken into. It doesn't really matter how strong the wall is, pickaxes etc do not break walls down.

**Comment by Ambrose997** [comment]
Why was auto assignment removed out of curiosity? It made things mega convenient for D-Class.

**Comment by Ambrose997** [comment]
re-add mirrors too please. Mirror loot was great.

**Comment by Ambrose997** [comment]
Kitchen is missing a rolling pin.

**Comment by Ambrose997** [comment]
D-Class want their cigarettes/lighters back >:( 

Also stasis gens + wrenches

**Comment by quadbium** [comment]
court
that courtroom has no witness stand

**Comment by ChromeFoxxity** [comment]
> court that courtroom has no witness stand

I'm pretty sure they could just use the normal stand, but fine.

**Comment by Ambrose997** [comment]
Can you cut it out with the fucking commit spam? 186 commits, 123 files changed, what the fuck? Leave my fucking inbox alone!

**Comment by WoodenTucker** [comment]
whats a comment spam

**Comment by WoodenTucker** [comment]
hi ambrose
image

**Comment by ChromeFoxxity** [comment]
> Can you cut it out with the fucking commit spam? 186 commits, 123 files changed, what the fuck? Leave my fucking inbox alone!

No.

**Pull request merged**